docs/src/interface.md

```diff
@@ -64,8 +64,12 @@ A decorator manifold extends the functionality of a [`Manifold`](@ref) in a semi
 It internally stores the [`Manifold`](@ref) it extends and by default for functions defined in the [`ManifoldsBase`](interface.md) it acts transparently in the sense that it passes all functions through to the base except those that it actually affects.
 For example, because the [`ArrayManifold`](@ref) affects nearly all functions, it overwrites nearly all functions, except a few like [`manifold_dimension`](@ref).
 On the other hand, the [`MetricManifold`](@ref) only affects functions that involve metrics, especially [`exp`](@ref) and [`log`](@ref) but not the [`manifold_dimension`](@ref).
+Contrary to the previous decorator, the [`MetricManifold`](@ref) does not overwrite functions.
+The decorator sets functions like [`exp`](@ref) and [`log`](@ref) to be implemented anew but required to be implemented when specifying a new metric.
+An exception is then the when a new metric [`is_default_metric`](@ref), which again makes the function transparent.
```

Comment:
```suggestion
An exception is when the new metric is a default metric (see [`is_default_metric`](@ref)), which again makes the function transparent.
```
I think it's more clear.

Reply:
Oh, that's still not a sentence. Maybe “An exception is not issued if a metric is additinally set to be the default metric (see [`is_default_metric`](@ref), since this makes all functions act transparently.“ ?

Reply:
Oh, that's still not a sentence. Maybe “An exception is not issued if a metric is additinally set to be the default metric (see is_default_metric, since this makes all functions act transparently.“ ?

Reply:
Yes, that even better.